account_move_restricted_group/models/account_move.py

```
@@ -0,0 +1,17 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+from odoo import models, fields
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def _get_action_move_out_invoice_type_group_evaluation(self):
+        action = self.env.ref('account.action_move_out_invoice_type').sudo().read()[0]
```

This code should raise an error for a normal user, because in v17 actions model access is restricted for almost everyone (but admin), if I'm not wrong.
You could use `_for_xml_id()` instead.